addons/website/static/src/systray_items/edit_website.xml

```diff
@@ -1,14 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 <t t-name="website.EditWebsiteSystray">
-    <div class="o_menu_systray_item o_edit_website_container d-none d-md-block" t-att-class="{ o_not_translatable: !translatable }">
-        <a href="#" class="h-100" accesskey="a" t-on-click="startEdit" t-att-class="{'btn btn-primary d-flex align-items-center ps-0 pe-2 rounded-0 border-0 fw-normal': !translatable, 'o_nav_entry': translatable}">
-            <span t-att-class="{'o_nav_entry' : !translatable}">
-                <span t-if="translatable" class="fa fa-pencil me-1"/>
-                <span t-else="" class="fa" t-att-class="{ 'fa-refresh fa-spin me-2': this.state.isLoading, 'fa-pencil me-1': !this.state.isLoading }"/>
-                <t t-esc="label"/>
+    <div class="o_menu_systray_item o_edit_website_container d-none d-md-flex">
+        <a t-if="!translatable" href="#" class="o-website-btn-custo-primary btn position-relative d-flex align-items-center rounded-0 border-0 px-3 is_full_height" accesskey="a" t-on-click="startEdit">
+            <span t-if="this.state.isLoading" class="position-absolute top-50 start-50 translate-middle">
+                <i class="fa fa-refresh fa-spin"/>
             </span>
+            <span t-out="label" t-attf-class="{{ this.state.isLoading ? 'opacity-0' : 'opacity-100' }}"/>
         </a>
+        <Dropdown t-else="">
+            <button class="o-website-dropdown-toggle-custo o-website-btn-custo-primary btn rounded-0 border-0 px-3 is_full_height" data-hotkey="a">
+                <span t-out="label"/>
+            </button>
+            <t t-set-slot="content">
+                <DropdownItem onSelected="() => this.startTranslate()" class="'o_translate_website_dropdown_item'">
+                    Translate (<t t-out="this.websiteService.currentWebsite.metadata.lang"/>)
+                </DropdownItem>
+                <DropdownItem onSelected="() => this.startEdit()" class="'o_edit_website_dropdown_item'">
+                    Edit (<t t-out="this.websiteService.currentWebsite.metadata.defaultLang"/>)
+                </DropdownItem>
```

Comment:
![image](https://github.com/odoo/odoo/assets/30048408/2e18b190-256e-4b4f-a8cc-4a12f286d0b3)

Lang codes are IMO not easy to understand (and a bit ugly), wouldn't it be better with their name?
![image](https://github.com/odoo/odoo/assets/30048408/c2312927-790e-4293-af4e-cb67e4552a45)

Maybe could be even nicer with some quick css win, probably not really needed tho
![image](https://github.com/odoo/odoo/assets/30048408/50c3782a-a3f1-4b1a-8fc7-c887f2d6126a)

Reply:
Yes, indeed it would be better 👍. Although by using only the name, there will be an issue when a website will use language “variants”, such as “English US” and “English AU“. The rendering will be
- “Edit (English)”
- “Translate (English)”.

Why not use the same rendering as the language selector? Something like this : 
![Capture d’écran 2024-03-07 à 11 29 58](https://github.com/odoo/odoo/assets/80679690/f0d2f2dd-ac29-4c6e-8fc7-a161b486d6a2)

If so, @SergeBayet do you think it's possible to achieve this result ? could you have a look at it ?
I can change the style afterwards 🙂

Reply:
That would be really better indeed

Reply:
I'm on it :)

Reply:
@rdeodoo @Brieuc-brd It's done. Could you have a look? thanks

Reply:
Thanks @SergeBayet, looks good ! 👍